include/Microtuner.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Microtuner.h - manage tuning and scale information of an instrument
+ *
+ * Copyright (c) 2020 Martin Pavelek <he29.HS/at/gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef MICROTUNER_H
+#define MICROTUNER_H
+
+#include "AutomatableModel.h"
+#include "ComboBoxModel.h"
+#include "JournallingObject.h"
+#include "lmms_constants.h"
+#include "Note.h"
+
+class InstrumentTrack;
+
+class LMMS_EXPORT Microtuner : public Model, public JournallingObject
+{
+	Q_OBJECT
+public:
+	explicit Microtuner(InstrumentTrack *parent);
+
+	bool enabled() const {return m_enabledModel.value();}
+	bool keyRangeImport() const {return m_keyRangeImportModel.value();}
```

Comment:
This function is always used with `enabled()`, so does it make sense to make a combined function for convenience?

Reply:
Nice observation. If Microtuner is disabled, custom keymaps are also disabled so the range import can (and should) simply report false.